dts/common/nordic/nrf54h20.dtsi

```diff
@@ -1143,6 +1134,20 @@
 		};
 	};
 
+	cpuppr_private: cpuppr-private-bus {
```

Comment:
I think private buses should be defined in CPU-specfic files. Would it be OK to move it to nrf54h20_cpuppr.dtsi?

Reply:
I see other PPBs are in this file.

For consistency let's keep it as it is proposed and maybe move them out as a separate task.

Reply:
they need global visibility to eg, indicate interrupt-parent on reserved peripherals